docs/topics/gradle/gradle-configure-project.md

```diff
@@ -37,12 +37,13 @@ plugins {
 </tab>
 </tabs>
 
-When configuring your project, check the Kotlin Gradle plugin compatibility with available Gradle versions:
+When configuring your project, check the Kotlin Gradle plugin compatibility with available Gradle versions. 
+In the following table, there are minimum and maximum fully supported versions of Gradle and Android Gradle plugin:
 
-|                       | Minimum supported version | Maximum fully supported version |
-|-----------------------|---------------------------|---------------------------------|
-| Gradle                | %minGradleVersion%        | %maxGradleVersion%              |   
-| Android Gradle plugin | %minAndroidGradleVersion% | %maxAndroidGradleVersion%       |
+| Kotlin version | Gradle min and max versions              | Android Gradle plugin min and max versions            |
+|----------------|------------------------------------------|-------------------------------------------------------|
+| 1.8.0          | %minGradleVersion% – %maxGradleVersion%  | %minAndroidGradleVersion% – %maxAndroidGradleVersion% |   
```

Comment:
will placeholder being replaced on actual merge?

Reply:
Sure, even not before the merge but soon, after a TeamCity successfully builds staging. The values are taken from here https://github.com/JetBrains/kotlin-web-site/pull/3333/files#diff-142a214d21f3012db045510fef72790293d95f0b2de8865cc16396783d723c0d